examples/abp_pcap_detection/run.py

```diff
@@ -136,16 +137,20 @@ def run_pipeline(
             filter_null=False,
         ))
 
-    # Add a deserialize stage
+    # Add a deserialize stage.
+    # This stage deserializes source data into Dataframes.
```

Comment:
I think this is no longer true, the source stage emits MessageMeta objects, and the DeserializeStage simply chunks them into smaller MultiMessage's according to the config's `pipeline_batch_size`